chgnet/model/dynamics.py

```diff
@@ -294,7 +295,7 @@ def relax(
                 cry_obs = CrystalFeasObserver(atoms)
 
             if relax_cell:
-                atoms = ase_filter(atoms)
+                atoms = FrechetCellFilter(atoms)
```

Comment:
i think this should stay `ase_filter`

Reply:
So we just ignore this following warning?

```
/Users/runner/work/chgnet/chgnet/chgnet/model/dynamics.py:297: DeprecationWarning: Use FrechetCellFilter for better convergence w.r.t. cell variables.

    atoms = ase_filter(atoms)
```

Reply:
`FrechetCellFilter` should be the default but there needs to be a way to customize it for people who want to compare with legacy behavior from last year when `FrechetCellFilter` didn't exist yet and `ExpCellFilter` was the default

Reply:
Sure reverted in aeb90530c51d80708031b66248430fc33d330c39